.changeset/ready-seas-ask.md

```diff
@@ -0,0 +1,18 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": deprecation
+---
+
+Deprecation Notice: Interfaces Migrated to `@fluidframework/core-interfaces`
```

Comment:
I think it would be nice for the title to be more specific, so someone just reading the titles can tell if it impacts them:

```suggestion
Deprecation Notice: Event Interfaces Migrated to `@fluidframework/core-interfaces`
```